com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -433,6 +433,7 @@ static void InitializeStackedCameraData(Camera baseCamera, UniversalAdditionalCa
             cameraData.targetTexture = baseCamera.targetTexture;
             cameraData.isStereoEnabled = IsStereoEnabled(baseCamera);
             cameraData.isSceneViewCamera = baseCamera.cameraType == CameraType.SceneView;
+            cameraData.isPreviewCamera = baseCamera.cameraType == CameraType.Preview;
```

Comment:
hm, we should probably add to cameraData the camera type isntead. otherwise we might be in a situation that we add every single type.

Reply:
@phi-lira Yeah, that's a good point. 
The question then is: Should we then remove or keep the isSceneViewCamera variable?